### PR TITLE
feat: Add webpack and bundle file

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
This is the basic implementation of this decision  https://github.com/nelc/frontend-essentials/pull/5

## How to test

1.  Check this branch
2. Run `npm run build-bundle`

The previous command should create a new file within the folder dist called  frontend-essentials.js, that will contain the exported components inside bundle.tsx

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+module.exports = {
+  mode: 'production',
+  entry: './src/index.tsx',
+  externals: {
+    react: 'React',
+    'react-dom': 'ReactDom',
+  },
+  output: {
+    filename: 'frontend-essentials.js',
```
Comment (johanv26) [inline webpack.config.js:11]: This could be called frontend-essentials-bundle.js
OR something related bundle

Comment (johanv26) [reply webpack.config.js:11]: The rest, the bundle is being created.

Comment (andrey-canon) [reply webpack.config.js:11]: That is not the name that you approved on the ADR

**Comment by andrey-canon** [comment]
https://github.com/nelc/frontend-essentials/pull/13

**Pull request closed without merging**